docs/source/onnx.rst

```diff
@@ -342,9 +342,15 @@ If the operator is an ATen operator (shows up in the TorchScript graph with the
   Make sure the function has the same name as the ATen function, which may be declared in
   ``torch/_C/_VariableFunctions.pyi`` or ``torch/nn/functional.pyi`` (these files are generated at
   build time, so will not appear in your checkout until you build PyTorch).
-* The first arg is always the ONNX graph that is being built for export.
+* The first arg is always the ONNX graph that is being built for export (except for advanced symbolic functions).
```

Comment:
I'd reword to
"By default, the first arg is the ONNX graph...". Don't mention the advanced case in this bullet point.